docs-client/src/lib/transports/transport.ts

```diff
@@ -47,8 +47,37 @@ export default abstract class Transport {
     return this.doSend(method, filledHeaders, bodyJson, endpointPath, queries);
   }
 
+  public curlPathAndBody(method: Method, body: string): [string, string] {
+    const endpoint = this.findDebugMimeTypeEndpoint(method);
+    if (!endpoint) {
```

Comment:
Probably no need to check this because `findDebugMimeTypeEndpoint` do not return `undefined`?

Reply:
👍